.openshift-ci/compatibility_test.py

```
@@ -18,6 +19,31 @@
     get_compatibility_test_tuples,
 )
 
+HELM_REPO_NAME = "tmp-srox-compat"
+HELM_CHART_URL_DEFAULT = "https://raw.githubusercontent.com/stackrox/helm-charts/main/opensource"
+
+
+def _setup_helm_repo():
+    """Add the helm chart repo once for all bash subprocess calls."""
+    url = os.environ.get("COMPAT_HELM_CHART_URL", HELM_CHART_URL_DEFAULT)
+    subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)